capstone/scripts/helpers.py

```diff
@@ -117,3 +126,13 @@ def parse_xml(xml):
         xml = xml.encode('utf8')
         
     return PyQuery(xml, parser='xml', namespaces=nsmap)
+
+def copy_file(from_path, to_path, from_storage=None, to_storage=None):
+    """
+        Copy contents of from_path to to_path, optionally using storages instead of filesystem open().
+    """
+    from_open = from_storage.open if from_storage else open
```

Comment:
I like this a lot!

Reply:
Actually, why do we have to open the file in order to copy from/to? Why not just use `shutil.copyfile`

Reply:
This is mostly for copies involving Django storages (which might be on local disk or S3 or SFTP or anywhere else) -- it's not necessary if both paths are on disk.

(In general we do want to be using Django storages any time we touch files, though, unless we're specifically dealing with temp files -- that lets the same code run in different kinds of deployments.)